demo/screens/EthereumQuestion.js

```diff
@@ -1,12 +1,13 @@
 import React from "react";
-import { Image, Button, StyleSheet, View, Text } from "react-native";
-import { createStackNavigator, createAppContainer } from "react-navigation";
+import { Button, StyleSheet, View, Text } from "react-native";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+import Colors from "../constants/Colors";
 
 export default class EthereumQuestion extends React.Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text style={styles.questionText}>{`Are you new to Ethereum?`}</Text>
+        <Text style={styles.questionText}>Are you new to Ethereum?</Text>
```

Comment:
Interestingly, the highlighting looks weird on GitHub with this removal

Reply:
Yeah.

Reply:
Oh, I think one other reason to keep it like you had it before is that one of the two platforms (I forget if it's iOS or Android) doesn't like it without this

Reply:
(I just added changing it back as a suggestion via the GitHub UI)